android/src/com/frostwire/android/gui/util/UIUtils.java

```diff
@@ -113,6 +114,10 @@ private static void showToastMessage(Context context, String message, int durati
         }
     }
 
+    public static void showShortMaterialMessage(View view, String message) {
```

Comment:
why not call this `showShortMessage`, and if you receive a `resId`, you don't need to call `getString` while using it